### PR TITLE
fix export_to_phy template_ind

### PR DESCRIPTION
**Pull request by llobetv**
Hi :)
If compute_pc_features was False templates_ind.py was wrong. 
Because it used pc_feature_ind to be calculated and that the new method for calculate template ind without pca_ind was wrong (work with template without all channels). 
So I use channel_list return when waveform are exctracted that appear to be the same as pc_feature_ind. 

I don't remove the old method l 1462 but maybe we can?

### CODE REVIEW

**Comment by alejoe91** [comment]
HI @llobetv 

I thought I was re-computing template_ind as it is computed in the `get_unit_waveforms`, but it's clearly some duplicate code. I think using `channel_list` as you suggest is better. Can you remove the final else? :)

File: spiketoolkit/postprocessing/postprocessing_tools.py [hunk 1]
```diff
@@ -1457,7 +1457,9 @@ def _get_phy_data(recording, sorting, compute_pc_features, max_channels_per_temp
         # waveforms, templates, and pc_scores are computed on the same channels
         if pc_feature_ind is not None:
             templates_ind = pc_feature_ind
-        else:
+        elif channel_list is not None:
+            templates_ind = np.array(channel_list)
+        else: # this part doesn't seems to work if template do not cover all channels, remove it?
```
Comment (alejoe91) [inline spiketoolkit/postprocessing/postprocessing_tools.py:1462]: yep just remove this. Using `channel_list` is more consistent

**Comment by llobetv** [comment]
I make the change and use channel_list first. 
However there is one issue which is if channel_list and pc_feature_ind are None, I don't know if it is possible, but in that case it would probably bug?

File: spiketoolkit/postprocessing/postprocessing_tools.py [hunk 2]
```diff
@@ -1494,29 +1495,7 @@ def _select_max_channels_from_waveforms(wf, recording, max_channels):
         max_channel_idxs = np.arange(recording.get_num_channels())
 
     return max_channel_idxs
-
-
-def _select_max_channels_from_templates(template, recording, max_channels):
```
Comment (alejoe91) [inline spiketoolkit/postprocessing/postprocessing_tools.py:1499]: can you leave this? I might need it for other functions :)

**Comment by llobetv** [comment]
I also remove _select_max_channels_from_templates() that seems to be created for this purpose. 
Is it ok?

**Comment by alejoe91** [comment]
That should never happen as `channel_list` is always computed by the `get_unit_waveforms`.

File: spiketoolkit/postprocessing/postprocessing_tools.py (hunk 2 above)
Comment (llobetv) [reply spiketoolkit/postprocessing/postprocessing_tools.py:1499]: ok

**Comment by alejoe91** [comment]
> I also remove _select_max_channels_from_templates() that seems to be created for this purpose.
> Is it ok?

Can you actually leave it for now? could be useful

**Comment by llobetv** [comment]
Ok, done, I just set templates_ind = np.array(channel_list) every times as it should always exist

**Comment by alejoe91** [comment]
Thanks! merging

**Pull request merged**